crates/torii/migrations/20230316154230_setup.sql

```diff
@@ -10,12 +10,14 @@ CREATE TABLE worlds (
     world_class_hash TEXT,
     executor_address TEXT,
     executor_class_hash TEXT,
+    -- TODO: Add executed_at timestamp once we have a way to get it
```

Comment:
I don't think we need that here. We should instead store a txn hash once that is supported